code/game/objects/structures/bedsheet_bin.dm

```diff
@@ -32,6 +32,12 @@ LINEN BINS
 	add_fingerprint(user)
 	return
 
+/obj/item/weapon/bedsheet/attackby(obj/item/I, mob/user)
+	if(istype(I, /obj/item/weapon/wirecutters) || istype(I, /obj/item/weapon/shard))

```

Comment:
What about knives? Shhouldn't they work?

And what about energy swords - you can cut cake with them, after all.